### PR TITLE
[TIMOB-20405] iOS: Calculate Ti.UI.Tab iconInsets internally

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/TIMOB-20405

### CODE REVIEW

File: iphone/Classes/TiUITabProxy.m [hunk 1]
```diff
@@ -571,7 +571,7 @@ -(void)updateTabBarItem
     
     if (iconInsets != nil) {
         if(UIEdgeInsetsEqualToEdgeInsets([TiUtils contentInsets:iconInsets], [ourItem imageInsets]) == NO) {
-            [ourItem setImageInsets:[TiUtils contentInsets:iconInsets]];
+            [ourItem setImageInsets:[self calcumateIconInsets:iconInsets]];
```
Comment (cheekiatng) [inline iphone/Classes/TiUITabProxy.m]: it's calculate not calcumate

File: iphone/Classes/TiUITabProxy.m [hunk 2]
```diff
@@ -579,6 +579,18 @@ -(void)updateTabBarItem
     [rootController setTabBarItem:ourItem];
 }
 
+-(UIEdgeInsets)calcumateIconInsets:(id)value
```
Comment (cheekiatng) [inline iphone/Classes/TiUITabProxy.m]: calculate

**Comment by hansemannn** [comment]
Adressed comments.

**Comment by cheekiatng** [comment]
APPROVED!

**Pull request merged**